api/Bluetooth.json

```diff
@@ -70,6 +70,41 @@
           "deprecated": false
         }
       },
+      "availabilitychanged_event": {
+        "__compat": {
+          "description": "<code>availabilitychanged</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Bluetooth#availabilitychanged",
+          "spec_url": "https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetooth-onavailabilitychanged",
+          "support": {
+            "chrome": {
+              "version_added": "78"
```

Comment:
This is part of https://chromestatus.com/feature/6229592081694720 - based on tests in https://beaufortfrancois.github.io/samples/web-bluetooth/availability.html

Reply:
This feature was removed in https://github.com/mdn/browser-compat-data/pull/21643, which links to a bug that confirms this feature was never implemented.

Reply:
That's odd, the tests appear to work. But then the whole framework is hard to test, so thanks.

Reply:
REmoved.